src/platform/backends/shared/base_virtual_machine_factory.h

```diff
@@ -44,6 +44,10 @@ class BaseVirtualMachineFactory : public VirtualMachineFactory
         return {};
     };
 
+    void prepare_networking(std::vector<NetworkInterface>& extra_interfaces) override
+    {
+    }
+
```

Comment:
Do we want this to be in the base? I wonder if we should be explicit about this being empty on a given platform?

Reply:
I wondered the same, but ended up deciding against duplicating code. Figured that's what inheritance/base class implementations are for. Notice this is still separate from the interface (pure base) class.

I actually once asked the same (for [this](https://github.com/canonical/multipass/blob/6d86dc451ee89d90cb881653307bb2c7c5894682/src/platform/backends/shared/base_virtual_machine_factory.h#L61) from @luis4a0 in a review but ended up backtracking and apologizing :sweat_smile: 

WDYT? Would a comment like `// only certain backends need to do anything to prepare networking` help?

I do have to comment out that unused param in any case.

Reply:
I suppose it's just a "unimplemented" vs. "default behaviour" decision. Will leave it to you :)

Reply:
I added the comment to hopefully make it a little clearer.